docs/synapse/adminguides/permissions.rstorm

```diff
@@ -0,0 +1,590 @@
+  has been implemented. That is, an action that was not originally subject to a permissions check now has one
+  (usually because of a need to explicitly **deny** that action to particular users or roles).
+  
+  If a previously unchecked action were added with "default deny", it would potentially break existing Synapse
+  deployments by suddenly blocking an action that had been previously allowed (ungated). In these circumstances
+  the new permission is given a "default allow" that can then be specifically denied if necessary.
+
+
+.. admin_global_perms:
+
+Global (Cortex) Permissions
+---------------------------
+
+Permissions in Synapse can be assigned at the global (Cortex) level, or to a specific Auth Gate (see
+:ref:`admin_authgate_perms`). To assign permissions to an Auth Gate, you must specify its identifier (iden)
+(i.e., using the ``--gate`` option to the appropriate Storm command) when adding the associated rule to a user
+or role.
+
+If you do not specify an Auth Gate, the permissions are **global** and apply to any / all instances within the
+Cortex where a user or role has access. For example, the following Storm command:
+
+::
+  
+  auth.role.addrule all node
+
+...grants (allows) the ``node`` permission to the built-in **all** role. This allows **any** user (because all
+users are granted the **all** role by default) to perform **any** action on **any** node in **any** layer that
+is the topmost (writeable) layer in **any** view that the user can see.
+
+Specifying rules at the global (Cortex) level may be sufficient for many basic Synapse deployments.
+
+.. NOTE::
+  
+  Recall that **order matters** when adding rules; each rule is added to the end of the list of rules assigned
+  to a user or role, and rules are evaluated in order of precedence. To reorder rules, you must remove and
+  re-add them in the new order (unless using the commercial Synapse UI, which allows drag-and-drop reordering).
+
+Assign Permissions
+~~~~~~~~~~~~~~~~~~
+
+Permissions rules (allow or deny) are assigned using the :ref:`storm-auth-user-addrule` and :ref:`storm-auth-role-addrule`
+commands.
+
+**Examples:**
+
+Prevent the user "ron" from setting tag descriptions (setting the ``syn:tag:desc`` property):
+
+.. storm-pre:: auth.user.add ron --email ronthecat@vertex.link
+.. storm-cli:: auth.user.addrule ron "!node.prop.set.syn:tag:desc"
+
+.. TIP::
+  
+  Deny rules specified with Storm must be enclosed in quotes (single or double) because they begin with
+  a symbol ( ! ).
+
+
+Allow the role "senior analysts" to add tags in threat attribution (``cno.threat``) tag tree:
+
+.. storm-pre:: auth.role.add "senior analysts"
+.. storm-cli:: auth.role.addrule "senior analysts" node.tag.add.cno.threat
+
+Prevent the "all" role from deleting nodes:
+
+.. storm-cli:: auth.role.addrule all "!node.del"
+
+.. TIP::
+  
+  You can view information about a user or role (including associated roles and / or rules) with the
```

Comment:
Instead of being a `tip` we could just make this a in-line directive for the user to run and see the output of, based on the previously executed commands?

Reply:
The commands are described /shown in the "managing users and roles" section - linked back to these in the revised doc. lmk if this works.